File: Assignment_11/Assignment_java/Assignment_5.java
import java.util.*;

class Pattern 
{
    public void Display(int iNo)
    {
        int iCnt = 0;
       
        for(iCnt = 1 ; iCnt <= iNo * 2 ; iCnt++)
        {
            if(iCnt % 2 == 0)
            {
                System.out.print(iCnt+"\t");
            }
            
        }
        System.out.println();
    }
}
class Assignment_5
{


    public static void main(String A[])
    {   
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);
    }
}

File: Assignment__11/Assignment_5.c

#include<stdio.h>
void Pattern(int iNo )
{
    int iCnt = 0;
    

    for(iCnt = 1 ; iCnt <= iNo*2 ; iCnt++)
    {
        if((iCnt % 2) == 0 )
        {
            printf("%d\t",iCnt); 
        }
       
    }
    printf("\n");
}
int main()
{
    int iValue = 0;
    printf("Enter number : ");
    scanf("%d",&iValue);
    Pattern(iValue);

    return 0 ;

}




